hw/ip/nmi_gen/data/nmi_gen.hjson

```diff
@@ -0,0 +1,38 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+# NMI_GEN register template
+{
+  name: "NMI_GEN",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  regwidth: "32",
+  param_list: [
```

Comment:
is `param_list` needed if it is an empty list?

Reply:
removed